robotpy_installer/pyproject.py

```diff
@@ -171,6 +175,308 @@ def write_default_pyproject(
     with open(toml_path(project_path), "w") as fp:
         fp.write(content)
 
+    ignore_content = inspect.cleandoc(
+        """
+    # Executables
+    *.exe
+    *.out
+    *.app
+
+    # Log file
+    *.log
+
+    # Package Files #
+    *.jar
+    *.war
+    *.nar
+    *.ear
+    *.zip
+    *.tar.gz
+    *.rar
+
+    # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+    hs_err_pid*
+
+    ### Linux ###
+    *~
+
+    # temporary files which can be created if a process still has a handle open of a deleted file
+    .fuse_hidden*
+
+    # KDE directory preferences
+    .directory
+
+    # Linux trash folder which might appear on any partition or disk
+    .Trash-*
+
+    # .nfs files are created when an open file is removed but is still being accessed
+    .nfs*
+
+    ### macOS ###
+    # General
+    .DS_Store
+    .AppleDouble
+    .LSOverride
+
+    # Icon must end with two \r
+    Icon
+
+    # Thumbnails
+    ._*
+
+    # Files that might appear in the root of a volume
+    .DocumentRevisions-V100
+    .fseventsd
+    .Spotlight-V100
+    .TemporaryItems
+    .Trashes
+    .VolumeIcon.icns
+    .com.apple.timemachine.donotpresent
+
+    # Directories potentially created on remote AFP share
+    .AppleDB
+    .AppleDesktop
+    Network Trash Folder
+    Temporary Items
+    .apdisk
+
+    ### VisualStudioCode ###
+    .vscode/*
+    !.vscode/settings.json
+    !.vscode/tasks.json
+    !.vscode/launch.json
+    !.vscode/extensions.json
+
+    ### Windows ###
+    # Windows thumbnail cache files
+    Thumbs.db
+    ehthumbs.db
+    ehthumbs_vista.db
+
+    # Dump file
+    *.stackdump
+
+    # Folder config file
+    [Dd]esktop.ini
+
+    # Recycle Bin used on file shares
+    $RECYCLE.BIN/
+
+    # Windows Installer files
+    *.cab
+    *.msi
+    *.msix
+    *.msm
+    *.msp
+
+    # Windows shortcuts
+    *.lnk
+
+    ### Gradle ###
```

Comment:
A significant chunk of these contents are C++/Java specific. There's no reason to include them for a Python project.